### ISSUE TITLE
Original Genres

### ISSUE THREAD

**Issue by sastoudt** [the-pudding/kidz-bop-data#5]
I first tried to get this information from Genius since we have artist IDs already, but I wasn't able to find it.

Genre is a bit buried in the Spotify API. First I have to get the discography of each artist by name just to access the artist ID. I have to figure out the rate limit on this step before I go full speed ahead. Then I can feed a bunch of artist IDs to get genre. However, it isn't a nice and simple genre. For example, for NSYNC it gives me: boy band, dance pop, europop, pop, post-teen pop. 

So maybe we should talk about how to proceed here. Is it easier to manually enter genre or find some way to programmatically synthesize detailed genres?

**Comment by jadiehm** [comment]
Hmmm... yeah this seems tough. I just looked into it a bit more and I'm wondering if we should drop because we really need the genre of each song instead of the artist. Taylor Swift's "Love Song" would fall under country, but her "ME!" would definitely be pop. I can see a lot of crossovers and genre blurring happing especially in songs that feature multiple artists (Maroon 5 + Cardi B "Girls Like You"). And I'm not sure the manual legwork would be worth the payoff.

**Comment by sastoudt** [comment]
Yeah, I think that makes sense. Let's forget it for now and we can revisit later, especially if we have a more targeted subset of interest from looking at other results.
